packages/gatsby-plugin-gatsby-cloud/src/components/IndicatorButton.js

```diff
@@ -114,7 +114,7 @@ const IndicatorButton = ({
         <div
```

Comment:
this should become a button if it's clickable for a11y

Reply:
Good call!